modules/Bio/EnsEMBL/IO/TabixParser.pm

```diff
@@ -49,7 +49,7 @@ sub open {
 
   my $self = $class->SUPER::new(@other_args);
 
-  confess "ERROR: Input file is not bgzipped, cannot use tabix\n" unless $filename =~ /\.gz$/;
+  confess "ERROR: Input file is not bgzipped, cannot use tabix\n" unless $filename =~ /\.gz$/ || $filename =~ /\.bgz$/;
```

Comment:
What do you thing of providing the filename in the error message, and updating the regex so it does not need the or? (it is not a complex one...)
`print "ERROR: Input file $filename is not bgzipped, cannot use tabix\n" unless $filename =~ /\.b?gz$/;`